src/ontology/ro-edit.owl

```diff
@@ -6147,6 +6148,15 @@ AnnotationAssertion(rdfs:label obo:RO_0016002 "has disease")
 AnnotationAssertion(rdfs:seeAlso obo:RO_0016002 "https://github.com/oborel/obo-relations/issues/478")
 SubObjectPropertyOf(obo:RO_0016002 obo:RO_0016001)
 
+# Object Property: obo:RO_0016004 (has exposure medium)
+
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0016004 "X has exposure medium Y if X is an exposure event (process), Y is a material entity, and the stimulus for X is transmitted or carried in Y.")
+AnnotationAssertion(obo:IAO_0000119 obo:RO_0016004 "ExO_0000083")
```

Comment:
This is just a string, and not even a CURIE

I would recommend instead annotating the definition axiom - or if the current annotation datamodel is used then use a URL instead

Reply:
I moved the 'definition source' annotation onto the definition itself, hopefully that is the change you were intending to see. Please let me know if not!